lib/pd.js

```diff
@@ -133,6 +133,72 @@ function Client(apiToken, tokenType, options) {
         }
     };
 
+// Analytics
+// https://developer.pagerduty.com/api-reference/reference/REST/openapiv3.json/paths/~1analytics~1raw~1incidents~1%7Bid%7D/get
+    this.analytics = {
+        getIncidentData: async (id) => {
+            try {
+                return await request(this.options('/analytics/raw/incidents/'+ id, 'GET', null, null))
+            }
+            catch (err) {
+                throw new Error(err)
+            }
+        },
+        getIncidentsData: async () => {
+            try {
+                return await request(this.options('/analytics/raw/incidents', 'GET', null, null))
+            }
+            catch (err) {
+                throw new Error(err)
+            }
+        },
+    }
```

Comment:
Can you remove the leading / from the options section `/analytics/raw/incidents`